seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -84,20 +86,33 @@ def test_create_project_from_workspace(self, mock_email, mock_add_service_accoun
                          .format(TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME))
 
         # Test the user needs sufficient workspace permissions
-        self.check_manager_login(url)
+        self.mock_get_ws_access_level.reset_mock(side_effect=True)
+        self.mock_get_ws_access_level.return_value = {
+            'accessLevel': 'READER',
+            'canShare': False,
+        }
+        self.login_collaborator()
+        _ = self.client.post(url)
         mock_utils_logger.warning.assert_called_with('test_user_collaborator does not have sufficient permissions for workspace {}/{}'
                                                .format(TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME))
 
         # Test missing required fields in the request body
+        self.login_manager()
+        self.mock_get_ws_access_level.return_value = {
+            'accessLevel': 'WRITER',
+            'canShare': True,
+            'workspace': {'bucketName': 'test_bucket'}
```

Comment:
I would strongly prefer you just use the main mocked fixtures for all the test to ensure consistent behavior. Theres no harm in justadding this to the test fixture data